ui2/yarn.lock

```diff
@@ -0,0 +1,41 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Oof, this file is in `ui2/` which is a directory that seems to have come back from the dead here! Not sure why that happened 😅 

Try a `git rm -rf ui2/` if you're actually seeing it locally

Reply:
oh man, okay not sure how this happened